src/clef.ts

```diff
@@ -200,9 +192,16 @@ export class Clef extends StaveModifier {
 
     this.applyStyle(ctx);
     ctx.openGroup('clef', this.getAttribute('id'));
-    Glyph.renderGlyph(ctx, this.x, stave.getYForLine(this.clef.line), this.clef.point, this.clef.code, {
-      category: `clef_${this.size}`,
-    });
+    Glyph.renderGlyph(
+      ctx,
+      this.x,
+      stave.getYForLine(this.clef.line),
+      this.size == 'default' ? Tables.NOTATION_FONT_SCALE : (Tables.NOTATION_FONT_SCALE / 3) * 2,
```

Comment:
Same: Can you add a comment here to explain why it is multiplied by 2/3 ?

Reply:
Done in `Clef.getPoint`